scripts/dpf-sanity-checks.sh

```
@@ -0,0 +1,346 @@
+
+testcase_title="Checking if any cluster operators are degraded or progressing on hosted cluster" 
+echo -e "\n${testcase_title}"
+check_cluster_operators "${hosted_kubecfg}"
+result_check_cluster_operators=$?
+test_results_summary+="\n${testcase_title}: $(format_result ${result_check_cluster_operators})"
+
+echo -e "\nfailed_testcase_count is: ${failed_testcase_count}"
+
+
+echo -e "\nChecking if workload namespace exists on admin cluster, otherwise create it"
+if oc get namespace "${sriov_test_pod_namespace}" --kubeconfig="${admin_kubecfg}" >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[warning] sriov_test_pod_namespace is referenced but not assigned.

(SC2154)